configure.ac

```diff
@@ -845,6 +845,14 @@ AC_CHECK_HEADERS([linux/fsverity.h],[FSVERITY_IOCTL="yes"])
 ])
 AM_CONDITIONAL(FSVERITY_IOCTL,[test "x$FSVERITY_IOCTL" = xyes])
 
+#=================
+# Check for fapolicyd support
+AC_ARG_WITH(fapolicyd,
+AS_HELP_STRING([--with-fapolicyd],[build with File Access Policy Daemon support]),
```

Comment:
`--with-fapolicyd` sounds like such a *daemon* will be built as a result. I dunno, I'd perhaps call this "fapolicy" (ie drop the trailing d) everywhere, or maybe there are better alternatives. Not that it's a showstopper or anything, names are hard.